src/active-dialog.c

```diff
@@ -135,42 +127,41 @@ start_proj (GObject *obj, GttActiveDialog *dlg)
 
 /* =========================================================== */
 
-static void
-setup_menus (GttActiveDialog *dlg)
+static void setup_menus (GttActiveDialog *dlg)
 {
 	GtkMenuShell *menushell;
 	GList *prjlist, *node;
-	char * msg;
+	char *msg;
 
-	msg = _("No project timer is currently running in GnoTime.  "
-	        "Do you want to start a project timer running?  "
-	        "If so, you can select a project from the menu below, "
-	        "and click 'Start' to start the project running.  "
-	        "Otherwise, just click 'Cancel' to do nothing.");
+	msg = _ ("No project timer is currently running in GnoTime.  "
+	         "Do you want to start a project timer running?  "
+	         "If so, you can select a project from the menu below, "
+	         "and click 'Start' to start the project running.  "
+	         "Otherwise, just click 'Cancel' to do nothing.");
 
 	gtk_label_set_text (dlg->active_label, msg);
 
-	msg = _("You can credit this project with the time that you worked "
-	        "on it but were away from the keyboard.  Enter a time below, "
-	        "the project will be credited when you click 'Start'");
+	msg = _ ("You can credit this project with the time that you worked "
+	         "on it but were away from the keyboard.  Enter a time below, "
+	         "the project will be credited when you click 'Start'");
```

Comment:
I don't object to space between function name and open parenthesis, and it's a common pattern. It looks like the existing code used a space in many cases, and also in many cases didn't.

It does look a little weird with this free-floating underscore. I can't quite recall having seen that before. :)

I wonder if clang can be told to skip the space for some definition of "short" function names?

Or if you would be happy without the space, that would be fine by me too.

Reply:
I had a quick look in some core gnome applications and it seems common to have space between before parentheses except when the thing before is a translation helper such as `_` or `C_`, e.g:

```c
return g_strdup (_("24-hour"));

label = g_strdup_printf (C_("timezone desc", "%s (%s)"), ...
```

It does not seem clearly spelt out in any written style guide I could find, and there does not seem to be a clang format defintion, so I guess it's just something that people do.

If there is no easy way to change it, I'm fine with just letting it be.

Reply:
I went with the easy way of dropping the space before parenthesis (except for control statements). I only configured it like this because to me it seem the preferred formatting so far in the codebase, but the `_ (` was bugging me too. So if we have unity on this let's get rid of the space before parenthesis.

Reply:
Yes, definitely keep them after control statements.